assets/src/legacy/map.js

```diff
@@ -4491,6 +4491,10 @@ $(document).ready(function () {
         .children('.ui-dialog-titlebar').removeClass('ui-corner-all');
     // configurate OpenLayers
     OpenLayers.DOTS_PER_INCH = 96;
+    const lizmapVariables = JSON.parse(document.getElementById('lizmap-vars').innerText);
```

Comment:
probably we should catch potential exception in the case of a bad JSON structure? And check if it is not empty?